diff --git a/lisp/international/mule.el b/lisp/international/mule.el
index 8978a97..f52de1e 100644
--- a/lisp/international/mule.el
+++ b/lisp/international/mule.el
@@ -312,10 +312,6 @@ Return t if file exists."
     (let ((buffer (generate-new-buffer " *load*"))
           (load-in-progress t)
           (source (string-suffix-p ".el" fullname)))
-      (unless nomessage
-	(if source
-	    (message "Loading %s (source)..." file)
-	  (message "Loading %s..." file)))
       (when purify-flag
 	(push (purecopy file) preloaded-file-list))
       (unwind-protect
@@ -359,11 +355,6 @@ Return t if file exists."
 	(let (kill-buffer-hook kill-buffer-query-functions)
 	  (kill-buffer buffer)))
       (do-after-load-evaluation fullname)
-
-      (unless (or nomessage noninteractive)
-	(if source
-	    (message "Loading %s (source)...done" file)
-	  (message "Loading %s...done" file)))
       t)))
 
 (defun charset-info (charset)
diff --git a/lisp/org/org.el b/lisp/org/org.el
index 4755bca..e9a9ba5 100644
--- a/lisp/org/org.el
+++ b/lisp/org/org.el
@@ -246,10 +246,8 @@ byte-compiled before it is loaded."
     (if compile
 	(progn
 	  (byte-compile-file tangled-file)
-	  (load tangled-file)
-	  (message "Compiled and loaded %s" tangled-file))
-      (load-file tangled-file)
-      (message "Loaded %s" tangled-file))))
+	  (load tangled-file))
+      (load-file tangled-file))))
 
 (defcustom org-babel-load-languages '((emacs-lisp . t))
   "Languages which can be evaluated in Org buffers.
diff --git a/src/lread.c b/src/lread.c
index d3e0a63..f35221a 100644
--- a/src/lread.c
+++ b/src/lread.c
@@ -1601,21 +1601,6 @@ Return t if the file exists and loads successfully.  */)
   prev_saved_doc_string = 0;
   prev_saved_doc_string_size = 0;
 
-  if (!noninteractive && (NILP (nomessage) || force_load_messages))
-    {
-      if (is_module)
-        message_with_string ("Loading %s (module)...done", file, 1);
-      else if (is_native_elisp)
-	message_with_string ("Loading %s (native compiled elisp)...done", file, 1);
-      else if (!compiled)
-	message_with_string ("Loading %s (source)...done", file, 1);
-      else if (newer)
-	message_with_string ("Loading %s (compiled; note, source file is newer)...done",
-		 file, 1);
-      else /* The typical case; compiled file newer than source file.  */
-	message_with_string ("Loading %s...done", file, 1);
-    }
-
   return Qt;
 }
 
